core/cap-0033.md

```diff
@@ -651,10 +717,71 @@ operations[5]:
 where `<FEE>`, `<SEQ_NUM>`, and `<TIME_BOUNDS>` should all be substituted
 appropriately.
 
+### Example: Revoke Sponsorship
+In this example, we demonstrate how a sponsorship can be revoked. Let `S`
+be the sponsoring account. Let `K` be the `LedgerKey` for an entry which
+is currently sponsored by `S`. Then the following transaction achieves the
+desired goal:
+
+```
+sourceAccount: S
+fee: <FEE>
+seqNum: <SEQ_NUM>
+timeBounds: <TIME_BOUNDS>
+memo:
+    type: MEMO_NONE
+operations[0]:
+    sourceAccount: S
+    body:
+        type: UPDATE_SPONSORSHIP
+        ledgerKey: K
+```
+
+where `<FEE>`, `<SEQ_NUM>`, and `<TIME_BOUNDS>` should all be substituted
+appropriately.
+
+### Example: Transfer Sponsorship
+In this example, we demonstrate how a sponsorship can be transferred. Let `S1`
+and `S2` be the sponsoring accounts. Let `K` be the `LedgerKey` for an entry
+which is currently sponsored by `S1`. Then the following transaction achieves
+the desired goal:
+
+```
+sourceAccount: S1
+fee: <FEE>
+seqNum: <SEQ_NUM>
+timeBounds: <TIME_BOUNDS>
+memo:
+    type: MEMO_NONE
+operations[0]:
+    sourceAccount: S2
+    body:
+        type: SPONSOR_FUTURE_RESERVES
+        sponsoredID: S1
```

Comment:
So when transferring sponsorship the sponsored account becomes the current sponsor? 🤯 This is very unintuitive to me.